python/ray/util/sgd/torch/distributed_torch_runner.py

```diff
@@ -142,14 +142,7 @@ def _get_model_state_dicts(self):
 
         This is needed for PyTorch DistributedDataParallel models.
         """
-        cpu_state_dicts = []
-        for model in self.models:
-            state_dict = model.module.state_dict()
-            # This is so that we create a duplicate of weights into CPU rather
-            # than move the model weights out of the GPU so that we can
-            # resume training while saving intermediate checkpoints.
-            cpu_state_dicts += [{k: v.cpu() for k, v in state_dict.items()}]
-        return cpu_state_dicts
```

Comment:
We removed this because you can now actually just use the state_dict from the Local Runner without copying it into CPU.